src/main/views/macros/form.njk

```diff
@@ -0,0 +1,14 @@
+{% macro submitButton(label = 'Continue', id = '', ariaLabel = '') %}

```

Comment:
should we name this file something other than `form`? it doesnt reflect what is in it, theres no form in here, maybe `common-components` or something along those lines?